File: CPP/well.cpp
#include <iostream>
using namespace std ;

#include "well.h"

void Well::render() {
	cout << "W" ;
}

int Well::getIdentity() {
	return 3 ;
}

void Well::Interact(int) {
	cout << "Air telah diisi penuh" << endl;
}

File: GUI-nya/Final/JavaGUI/src/facilities/Well.java
package facilities;

public class Well extends Facility {
	public char render(){
		return 'W' ; //System.out.print("W");
	}
	
	public int getIdentity(){
		return 3;
	}
	
	public void interact(int i){
		System.out.println("Air telah diisi penuh");
	}
};
